fury/tests/test_actors.py

```diff
@@ -154,6 +154,39 @@ def test_slicer():
                            np.array(slicer.shape))
 
 
+@npt.dec.skipif(skip_it)
+@xvfb_it
+def test_surface():
+    import random
+    # from scipy.spatial import Delaunay
+    import math
+    size = 11
+    vertices = list()
+    for i in range(-size, size):
+        for j in range(-size, size):
+            fact1 = - math.sin(i) * math.cos(j)
+            fact2 = - math.exp(abs(1 - math.sqrt(i ** 2 + j ** 2) / math.pi))
+            z_coord = -abs(fact1 * fact2)
+            vertices.append([i, j, z_coord])
+
+    random.shuffle(vertices)
+    vertices = np.array(vertices)
+    # xy = list()
+    # for coordinate in vertices:
+    #     xy.append([coordinate[0], coordinate[1]])
+    # tri = Delaunay(xy)
+    # faces = tri.simplices
+    renderer = window.renderer(background=(1, 1, 1))
+    surface_actor = surface(vertices, smooth="butterfly")
```

Comment:
you error come from here @PDN96. this is `actor.surface`  and not `surface` only.
if you want to use `surface` only, you have to import it: `from fury.actor import surface`